core/nodejs14MultitenantAction/supervisord.conf

```diff
@@ -0,0 +1,12 @@
+[supervisord]
+nodaemon=true
+
+[program:nodeServer]
+directory=/nodejsAction
+command=node --expose-gc app.js 
+autostart=true
+autorestart=true
+user=root
+startsecs=10
```

Comment:
I think we can go with `1` instead of `10`. wdyt ?

Reply:
fixed